src/index.scss

```diff
@@ -1,5 +1,5 @@
+
 $spacer: 1rem;
-$blue: #007db8;
```

Comment:
I'm trying to think through whether or not removing this is non-breaking.

While `$blue` [isn't being used anywhere in this repo](https://github.com/search?q=repo%3Aopenedx%2Ffrontend-component-header%20%24blue&type=code), it is possible someone has imported `index.scss` from this package somewhere and is using this `$blue`. I wouldn't consider that to be a *supported* scenario, but I also think just keeping `$blue` defined in here and waiting for the follow-up to remove it is the safest option.